packages/www/pages/_app.tsx

```diff
@@ -54,6 +56,10 @@ Object.keys(themes).map(
   (key, _index) => (themeMap[themes[key].className] = themes[key].className)
 );
 
+const livepeerClient = createReactClient({
+  provider: studioProvider({ apiKey: "b05094e0-05b6-4260-b462-bae581fdcf13" }), // TODO: replace with env var
```

Comment:
Do we really need any custom API key here? The `playbackInfo` API is public, and even if it wasn't, we should not need any API key to use the player receiving only an HLS URL.

We can very likely just leave the default API key here in case the player just needs some `LivepeerProvider` to be present (ideally it should not). Worst case, if we ever do need to call APIs with `livepeer.js`, we should not use an API key but actually the same user JWT that is used on the dashboard (check `ApiProvider`). This would give access to the account the user is logged in, not a random fixed account that we insert an API key here (worse, we'd likely need to put an **admin** API key here and we definitely don't want to leak that anywhere, ever).

Reply:
Ok I see what you're saying. Let me check if I can successfully initialise the provider without the api key

Reply:
I confirm that we can initialise it without any api key and there are not warnings or errors. So I will follow that

Reply:
Yeah, we shouldn't need an API key here to just use the player. If we are planning on using any other hooks from livepeer.js, we'd need to allow the client to provide a JWT. This might be a good feature though!

Reply:
Yeah this would be a nice feature. I think it'd actually be worthwhile refactoring the frontend to use livepeer.js at some point.